examples/list_interfaces.rs

```diff
@@ -10,16 +10,46 @@
 
 extern crate pnet_datalink;
 
+fn get_flags(interface: &pnet_datalink::NetworkInterface) -> String {
+    let mut flags: Vec<String> = vec![];
+    if interface.is_loopback() {
+        flags.push(String::from("LOOPBACK"));
+    }
+    if cfg!(any(target_os = "macos", target_os = "linux")) {
```

Comment:
I'd rather not have this behind a `cfg!()` - these methods should work on all OSes.